src/pls/data/nerd_icons.yml

```
@@ -5,31 +5,31 @@ nerd_icons:
   pls:        "\uf444"  # nf-oct-primitive_dot
 
   apple:      "\ue711"  # nf-dev-apple
-  audio:      "\ufc1d"  # nf-mdi-speaker_wireless
-  broom:      "\uf5e1"  # nf-mdi-broom
+  audio:      "\uf1c7"  # nf-fa-file_audio_o
+  broom:      "\ue615"  # nf-seti-config
   config:     "\ue615"  # nf-seti-config
   css:        "\ue749"  # nf-dev-css3
   docker:     "\uf308"  # nf-linux-docker
   folder:     "\uf07b"  # nf-fa-folder
-  git:        "\uf7a1"  # nf-mdi-git
+  git:        "\uf47f"  # nf-oct-git_compare
   github:     "\uf408"  # nf-oct-mark_github
   html:       "\ue736"  # nf-dev-html5
-  image:      "\uf7e8"  # nf-mdi-image
+  image:      "\uf1c5"  # nf-fa-file_image_o
   javascript: "\ue60c"  # nf-seti-javascript
   json:       "\ue60b"  # nf-seti-json
   license:    "\uf495"  # nf-oct-law
   lock:       "\uf456"  # nf-oct-lock
   markdown:   "\uf48a"  # nf-oct-markdown
   package:    "\uf487"  # nf-oct-package
   env:        "\ue22f"  # nf-fae-plant
-  python:     "\uf81f"  # nf-mdi-language_python
+  python:     "\ue73c"  # nf-dev-python
   ruby:       "\ue791"  # nf-dev-ruby_rough
-  runner:     "\ufc0c"  # nf-mdi-run
+  runner:     "\ueb7b"  # nf-cod-vm_running
   sass:       "\ue603"  # nf-seti-sass
-  shell:      "\ufcb5"  # nf-mdi-console_line
-  source:     "\uf668"  # nf-mdi-code_braces
-  test:       "\ufb67"  # nf-mdi-test_tube
+  shell:      "\uf489"  # nf-oct-terminal
+  source:     "\uf1c9"  # nf-fa-file_code_o
```

Seti's JSON braces are closer to the previous braces icon.
```suggestion
  source:     "\ue60b"  # nf-seti-json
```